widget/embedded/src/types/config.ts

```diff
@@ -21,6 +21,14 @@ export type WidgetColors = {
   info?: string;
 };
 
+export enum DisabledFeatures {
+  Notifications = 'notifications',
+  Theme = 'theme',
+  Liquidity_Sources = 'liquiditySources',
+  Connect_Wallet = 'connectWallet',
+  Language = 'language',
+}
+
```

Comment:
it is a common practice to use all caps for TypeScript enums.

Reply:
fixed.